platforms/hyperledger-besu/charts/node_validator/values.yaml

```diff
@@ -113,6 +113,11 @@ node:
     # Eg. ws: 8546
     ws: 
 
+# Enable/disable onchain permissioniong for besu nodes. 
+# Ref https://besu.hyperledger.org/en/stable/private-networks/tutorials/permissioning/onchain/
+permissioning:
```

Comment:
Please provide example value of the enabled field. (For reference, check how `ws` field is explained)
Also, as I asked earlier, this should be put in the logical place. This should go under node.permissioning section as we are enabling/disabling this field for the besu node. So it should be like `node.permissioning: enabled/disabled` (with the example as well for this key:value pair)

Reply:
this is added here  so that we can support offchain permissioning as well, we can have `type` as onchain/offchain permissioning

Reply:
this is added here  so that we can support offchain permissioning as well, we can have `type` as onchain/offchain permissioning